src/containers/Downloads/DownloadBar.js

```diff
@@ -19,7 +19,9 @@ let DownloadBar = ({ startDownload }) => {
         {() => (
           <div>
             <h1 className="share-link-modal__title">Sharable Link</h1>
-            <InputCopy value="Url to be copied, connect redux" />
+            <InputCopy
+              value={`${window.location.origin}/download/${dataSetId}`}
```

Comment:
Actually, it might be nice to store the domain in a constants file. And we could have one for `DEV` and one for `PROD`.

Reply:
hmm I'm not sure about this, because it will increase the complexity to deploy the project. 

We'll need one value for `STAGING`, and if someone runs the project locally in a different domain than `localhost:3000` they it will need to update the configuration file. What advantages you see in using a constant over this approach??

Reply:
Yeah, I see your point. We can stick with the `window.location` approach 👍

Reply:
what if we extract that code into a helper? `helpers.getDomain` or something like that. That way it will be easier if in the future we introduce a constant it will be easier to change

Reply:
Good call. Yeah, that would probably be helpful because I think we might have to introduce environment variables eventually anyways because the api domains will need to be based on the environment the app is in.